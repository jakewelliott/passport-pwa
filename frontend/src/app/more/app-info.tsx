import { PassportHeader } from '../../components/common/passport-header';
import { ContactUs } from '../../components/tabs/more/contact-us';
import { DownloadParkMaps } from '../../components/tabs/more/download-park-maps';
import { Reservations } from '../../components/tabs/more/reservations';

export const AppInfo = () => {
  return (
    <div className='max-w-[500px] mx-auto'>
      <div className='my-3.5'>
        <PassportHeader />
      </div>
      <ContactUs />
      <DownloadParkMaps />
      <Reservations />
    </div>
  );
};
