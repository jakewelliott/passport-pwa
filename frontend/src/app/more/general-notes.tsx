import { NotesMiniTab } from '../locations/components/notes-minitab';

export const EditGeneralNotes = () => {
	return (
		<div className='container mx-auto'>
			<div className='flex h-[calc(100vh-1.5rem-100px)] flex-col'>
				<NotesMiniTab parkId={0} />
			</div>
		</div>
	);
};
