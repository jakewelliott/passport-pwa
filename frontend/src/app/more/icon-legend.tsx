import { LegendSections } from '@/app/more/components/legend-sections';
import { dbg } from '@/lib/debug';

export const IconLegend = () => {
    dbg('RENDER', '/more/icon-legend');
    return (
        <div className='flex flex-col gap-4'>
            <h2 className='text-center'>Legend</h2>
            <LegendSections />
            <p className='min-w-[200px]'>
                <span className='text-icon-amenities'>Kids TRACK Trail: </span>Kids in Parks is a group that creates
                outdoor adventures called TRACK trails for families all over the country. Each one has fun things to
                learn about nature as you walk. You can even earn prizes by keeping track of your adventures! Check out
                kidsinparks.com to find out more.
            </p>
        </div>
    );
};
