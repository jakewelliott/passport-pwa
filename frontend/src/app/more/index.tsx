import RoundedButton from "../../components/common/rounded-button";

export default function More() {
	return (
		<>
			<h1>More</h1>
			<RoundedButton title="TEST" color="system_black" />
		</>
	);
}
