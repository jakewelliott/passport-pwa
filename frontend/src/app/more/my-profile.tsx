import { LoadingPlaceholder } from '@/components/loading-placeholder';
import RoundedButton from '@/components/rounded-button';
import { useUser } from '@/hooks/queries/useUser';
import { dbg } from '@/lib/debug';
import type { UserProfile } from '@/types';
import { useNavigate } from 'react-router-dom';

const AccountInfo = ({ user }: { user: UserProfile }) => {
    return (
        <div>
            <p>Username: {user.username}</p>
            <p>Role: {user.role}</p>
            {/* <p>Account created on: {DateHelper.toStringLong(user.createdAt)}</p> */}
        </div>
    );
};

export default function MyProfileScreen() {
    dbg('RENDER', '/more/my-profile');

    const { data: user, isLoading } = useUser();
    const navigate = useNavigate();

    const handleLogout = () => {
        const popup = window.confirm('Are you sure you want to logout?');
        if (popup) {
            navigate('/logout');
        }
    };

    const handlePasswordChange = () => {
        // TODO: implement password change
    };

    if (isLoading) return <LoadingPlaceholder what='your profile' />;

    return (
        <div className='flex flex-col gap-4'>
            <h2>My Profile</h2>
            {user && <AccountInfo user={user} />}
            <RoundedButton onClick={handlePasswordChange} title='Change Password' />
            <RoundedButton onClick={handleLogout} title='Logout' />
        </div>
    );
}
