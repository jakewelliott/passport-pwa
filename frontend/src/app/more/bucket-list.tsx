import { BucketListItem } from '@/components/tabs/more/bucket-list-item';

export const BucketList = () => {
  return (
    <div className='p-7'>
      <div className='max-w-96 mx-auto m-7'>
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
        <BucketListItem />
      </div>
    </div >
  );
};
