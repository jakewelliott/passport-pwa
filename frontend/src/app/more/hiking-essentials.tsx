import { HikingEssentialItem } from '@/components/tabs/more/hiking-essential-item';

export const HikingEssentials = () => {
  return (
    <div className='mx-auto max-w-4xl p-6' lang='en'>
      <h3 className='mb-5 bg-supporting_inactiveblue p-3 text-center text-system_white uppercase'>
        WHAT TO PACK FOR A HIKE
      </h3>

      <div className='flex flex-wrap justify-center gap-6'>
        <HikingEssentialItem
          imageSrc='/icons/hiking/boot.svg'
          altText='Hiking boot'
          title='HIKING BOOTS'
          description='Make sure your boots are broken in before hitting the trails to prevent blisters and sore feet!'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/water-bottle.svg'
          altText='Water bottle'
          title='WATER'
          description="You need to drink about 1 liter of water per every 4-5 miles hiked. Don't forget water for your pet!"
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/map-and-compass.svg'
          altText='Map and compass'
          title='MAP & COMPASS'
          description="Pick up a park map from the park's visitor center or download a digital version at ncparks.gov"
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/snacks.svg'
          altText='Snacks'
          title='SNACKS'
          description='You can make your own trail mix with dark chocolate, almonds, raisins and unsalted popcorn!'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/rain-gear.svg'
          altText='Rain gear'
          title='RAIN GEAR'
          description='When hiking in wet weather, avoid wearing cotton - it traps water and takes a long time to dry.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/flashlight.svg'
          altText='Flashlight'
          title='FLASHLIGHT'
          description='Due to all the trees, the trails will get darker before the rest of the park.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/first-aid-kit.svg'
          altText='First aid kit'
          title='FIRST AID KIT'
          description='Make sure your kit has bandages, antiseptic, antihistamines, and tweezers.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/matches.svg'
          altText='Matches'
          title='MATCHES'
          description='Pack waterproof matches or carry them in a waterproof container.'
        />

        <HikingEssentialItem
          imageSrc='/icons/hiking/knife.svg'
          altText='Knife'
          title='KNIFE'
          description='Make sure your knife or multitool has a locking blade to prevent any injuries.'
        />
      </div>
    </div>
  );
};
