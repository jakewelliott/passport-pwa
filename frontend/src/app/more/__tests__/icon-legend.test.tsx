import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { IconLegend } from '@/app/more/icon-legend';

describe('IconLegend Component - User Stories', () => {
	// User Story: As a user, I want to understand what each icon means
	it('should display icons with their meanings', () => {
		render(
			<BrowserRouter>
				<IconLegend />
			</BrowserRouter>
		);

		expect(screen.getByText('Icon Legend')).toBeInTheDocument();
		expect(screen.getByText('Activities')).toBeInTheDocument();
		expect(screen.getByText('Amenities')).toBeInTheDocument();
	});

	// User Story: As a user, I want to see camping options explained
	it('should explain camping options', () => {
		render(
			<BrowserRouter>
				<IconLegend />
			</BrowserRouter>
		);

		expect(screen.getByText('Tent camping')).toBeInTheDocument();
		expect(screen.getByText('RV hookup camping*')).toBeInTheDocument();
		expect(screen.getByText('Group camping')).toBeInTheDocument();
	});

	// User Story: As a user, I want to understand facility amenities
	it('should explain facility amenity icons', () => {
		render(
			<BrowserRouter>
				<IconLegend />
			</BrowserRouter>
		);

		expect(screen.getByText('Boat ramp')).toBeInTheDocument();
		expect(screen.getByText('Visitor center')).toBeInTheDocument();
		expect(screen.getByText('Playground')).toBeInTheDocument();
	});

	// User Story: As a user, I want to understand activity icons
	it('should explain activity icons', () => {
		render(
			<BrowserRouter>
				<IconLegend />
			</BrowserRouter>
		);

		expect(screen.getByText('Hiking')).toBeInTheDocument();
		expect(screen.getByText('Swimming')).toBeInTheDocument();
		expect(screen.getByText('Biking')).toBeInTheDocument();
	});

	// User Story: As a user, I want to understand state trail stamps
	it('should explain state trail stamps', () => {
		render(
			<BrowserRouter>
				<IconLegend />
			</BrowserRouter>
		);

		expect(screen.getByText('State Trail Stamps')).toBeInTheDocument();
		expect(screen.getByText('Mountains-to-Sea')).toBeInTheDocument();
		expect(screen.getByText('Fonta Flora')).toBeInTheDocument();
	});
}); 