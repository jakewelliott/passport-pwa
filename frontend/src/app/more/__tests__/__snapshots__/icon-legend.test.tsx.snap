// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`IconLegend Component - User Stories > matches snapshot 1`] = `
<div>
  <div
    class="flex flex-col gap-4"
  >
    <h2
      class="text-center"
    >
      Legend
    </h2>
    <div
      class="flex flex-col gap-4"
    >
      <div>
        <h4
          class="pb-3 text-icon_camping"
        >
          Camping
        </h4>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Camper Cabins"
            height="36px"
            src="/icons/park/CamperCabins-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Camper Cabins
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Camping"
            height="36px"
            src="/icons/park/Camping-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Canoe-in Camping"
            height="36px"
            src="/icons/park/CanoeinCamping-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Canoe-in Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Equestrian Camping"
            height="36px"
            src="/icons/park/EquestrianCamping-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Equestrian Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Group Cabins"
            height="36px"
            src="/icons/park/GroupCabins-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Group Cabins
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Group Camping"
            height="36px"
            src="/icons/park/GroupCamp-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Group Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Primitive Cabin"
            height="36px"
            src="/icons/park/PrimitiveCabin-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Primitive Cabin
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="RV Camping"
            height="36px"
            src="/icons/park/RVCamping-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              RV Camping
            </p>
            <p
              class="text-xs"
            >
              *
              has electric, water, AND/OR sewer hookups. Check with reservations website for available hookups
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Vacation Cabin"
            height="36px"
            src="/icons/park/VacationCabin-Green.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Vacation Cabin
            </p>
          </div>
        </div>
      </div>
      <div>
        <h4
          class="pb-3 text-icon_activities"
        >
          Activities
        </h4>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Biking"
            height="36px"
            src="/icons/park/Biking-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Biking
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Fishing"
            height="36px"
            src="/icons/park/Fishing-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Fishing
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Hiking"
            height="36px"
            src="/icons/park/Hiking-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Hiking
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Horseback Riding"
            height="36px"
            src="/icons/park/HorsebackRiding-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Horseback Riding
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Paddling"
            height="36px"
            src="/icons/park/Paddling-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Paddling
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Picnicking"
            height="36px"
            src="/icons/park/Picnicking-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Picnicking
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Rock Climbing"
            height="36px"
            src="/icons/park/RockClimbing-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Rock Climbing
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Swimming"
            height="36px"
            src="/icons/park/Swimming-Red.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Swimming
            </p>
          </div>
        </div>
      </div>
      <div>
        <h4
          class="pb-3 text-icon_amenities"
        >
          Amenities
        </h4>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Boat Ramp"
            height="36px"
            src="/icons/park/BoatRamp-Blue.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Boat Ramp
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Boat Rental"
            height="36px"
            src="/icons/park/BoatRental-Blue.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Boat Rental
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Exhibits"
            height="36px"
            src="/icons/park/Exhibits-Blue.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Exhibits
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Playground"
            height="36px"
            src="/icons/park/Playground-Blue.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Playground
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Visitor Center"
            height="36px"
            src="/icons/park/VisitorCenter-Blue.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Visitor Center
            </p>
          </div>
        </div>
      </div>
      <div>
        <h4
          class="pb-3 text-icon_state trail stamps"
        >
          State Trail Stamps
        </h4>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="French Broad State Trail"
            height="36px"
            src="/icons/park/FBST-Blaze.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              French Broad State Trail
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Fontana Lake State Trail"
            height="36px"
            src="/icons/park/FFST-Blaze.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Fontana Lake State Trail
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Hickory Nut Gorge State Trail"
            height="36px"
            src="/icons/park/HGST-Blaze.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Hickory Nut Gorge State Trail
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Mountains-to-Sea State Trail"
            height="36px"
            src="/icons/park/MST-Blaze.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Mountains-to-Sea State Trail
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Yadkin River State Trail"
            height="36px"
            src="/icons/park/YRST-Blaze.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Yadkin River State Trail
            </p>
          </div>
        </div>
      </div>
      <div>
        <h4
          class="pb-3 text-icon_trail amenities"
        >
          Trail Amenities
        </h4>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="4WD Beach Access"
            height="36px"
            src="/icons/misc/4WDBeach.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              4WD Beach Access
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Accessible Facilities"
            height="36px"
            src="/icons/misc/Accessible.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Accessible Facilities
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Amphitheater"
            height="36px"
            src="/icons/misc/Amphiteater.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Amphitheater
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Backpack Camping"
            height="36px"
            src="/icons/misc/BackpackCamping.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Backpack Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Bathhouse"
            height="36px"
            src="/icons/misc/Bathhouse.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Bathhouse
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Biking Trails"
            height="36px"
            src="/icons/misc/Biking.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Biking Trails
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Boat Ramp"
            height="36px"
            src="/icons/misc/BoatRamp.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Boat Ramp
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Boating Activities"
            height="36px"
            src="/icons/misc/Boating.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Boating Activities
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Camper Cabin"
            height="36px"
            src="/icons/misc/CamperCabin.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Camper Cabin
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Camping Area"
            height="36px"
            src="/icons/misc/Camping.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Camping Area
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Dump Station"
            height="36px"
            src="/icons/misc/DumpStation.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Dump Station
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Electric Hookup"
            height="36px"
            src="/icons/misc/ElectricHookup.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Electric Hookup
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Electric and Water Hookups"
            height="36px"
            src="/icons/misc/ElectricWaterHookups.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Electric and Water Hookups
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Equestrian Camping"
            height="36px"
            src="/icons/misc/EquestrianCamping.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Equestrian Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Firewood Available"
            height="36px"
            src="/icons/misc/Firewood.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Firewood Available
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Fishing Opportunities"
            height="36px"
            src="/icons/misc/Fishing.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Fishing Opportunities
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Group Camping Area"
            height="36px"
            src="/icons/misc/GroupCamp.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Group Camping Area
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Hiking Trails"
            height="36px"
            src="/icons/misc/Hiking.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Hiking Trails
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Horse Trailer Parking"
            height="36px"
            src="/icons/misc/HorseTrailerParking.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Horse Trailer Parking
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Horseback Riding"
            height="36px"
            src="/icons/misc/HorsebackRiding.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Horseback Riding
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Nearby Hospital"
            height="36px"
            src="/icons/misc/Hospital.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Nearby Hospital
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="IE Exhibits"
            height="36px"
            src="/icons/misc/IE_Exhibits.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              IE Exhibits
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Information Center"
            height="36px"
            src="/icons/misc/Information.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Information Center
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Marina Facilities"
            height="36px"
            src="/icons/misc/Marina.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Marina Facilities
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Paddle-In Camping"
            height="36px"
            src="/icons/misc/PaddleInCamping.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Paddle-In Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Paddling"
            height="36px"
            src="/icons/misc/Paddling.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Paddling
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Park Gate"
            height="36px"
            src="/icons/misc/ParkGate.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Park Gate
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Picnic"
            height="36px"
            src="/icons/misc/Picnic.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Picnic
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Picnic Shelter"
            height="36px"
            src="/icons/misc/PicnicShelter.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Picnic Shelter
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Point of Interest"
            height="36px"
            src="/icons/misc/PointofInterest.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Point of Interest
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Primitive Cabin"
            height="36px"
            src="/icons/misc/PrimitiveCabin.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Primitive Cabin
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Restroom"
            height="36px"
            src="/icons/misc/Restroom.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Restroom
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Rock Climbing"
            height="36px"
            src="/icons/misc/RockClimbing.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Rock Climbing
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Sewer Hookup"
            height="36px"
            src="/icons/misc/SewerHookup.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Sewer Hookup
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Swimming"
            height="36px"
            src="/icons/misc/Swimming.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Swimming
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="TRACK Trail"
            height="36px"
            src="/icons/misc/TRACKTrail.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              TRACK Trail
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Tent and Trailer Camping"
            height="36px"
            src="/icons/misc/TentTrailerCamping.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Tent and Trailer Camping
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Vacation Cabin"
            height="36px"
            src="/icons/misc/VacationCabin.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Vacation Cabin
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Viewing Area"
            height="36px"
            src="/icons/misc/ViewingSymbol.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Viewing Area
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Visitor Center"
            height="36px"
            src="/icons/misc/VisitorCenter.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Visitor Center
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Water Hookup"
            height="36px"
            src="/icons/misc/WaterHookup.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Water Hookup
            </p>
          </div>
        </div>
        <div
          class="my-1 flex items-center"
        >
          <img
            alt="Water Spigot"
            height="36px"
            src="/icons/misc/WaterSpigot.svg"
            width="36px"
          />
          <div
            class="ml-2 flex w-full flex-col justify-center"
          >
            <p>
              Water Spigot
            </p>
          </div>
        </div>
      </div>
    </div>
    <p
      class="min-w-[200px]"
    >
      <span
        class="text-icon-amenities"
      >
        Kids TRACK Trail: 
      </span>
      Kids in Parks is a group that creates outdoor adventures called TRACK trails for families all over the country. Each one has fun things to learn about nature as you walk. You can even earn prizes by keeping track of your adventures! Check out kidsinparks.com to find out more.
    </p>
  </div>
</div>
`;
